src/DssSpell.sol

```diff
@@ -18,50 +18,96 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import { MCD, DssInstance } from "dss-test/MCD.sol";
+import { FlapperInit, FlapperInstance, FlapperUniV2Config } from "src/dependencies/dss-flappers/FlapperInit.sol";
 
-interface RwaLiquidationLike {
-    function ilks(bytes32) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
-    function init(bytes32 ilk, uint256 val, string memory doc, uint48 tau) external;
-    function bump(bytes32 ilk, uint256 val) external;
+interface DssCronSequencerLike {
+    function addJob(address job) external;
 }
 
 interface RwaOutputConduitLike {
+    function deny(address usr) external;
     function hope(address usr) external;
     function nope(address usr) external;
     function mate(address usr) external;
     function hate(address usr) external;
     function kiss(address who) external;
+    function diss(address who) external;
     function file(bytes32 what, address data) external;
+    function clap(address _psm) external;
 }
 
 interface RwaUrnLike {
     function file(bytes32 what, address data) external;
 }
 
-interface NetworkPaymentAdapterLike {
-    function file(bytes32 what, address data) external;
+interface ChainlogLike {
+    function removeAddress(bytes32 _key) external;
+}
+
+interface VestLike {
```

Comment:
Sorry to nit you here but the pattern is `DssVestLike` per the archives.

`DssVestLike immutable MCD_VEST_DAI = DssVestLike(DssExecLib.getChangelogAddress("MCD_VEST_DAI"));`